src/site/stages/build/drupal/graphql/GetLatestPageById.graphql.js

```diff
@@ -1,11 +1,14 @@
 const landingPage = require('./landingPage.graphql');
 const page = require('./page.graphql');
-const fragments = require('./fragments.graphql');
 const healthCareRegionPage = require('./healthCareRegionPage.graphql');
-const newsStoryPage = require('./newStoryPage.graphql');
 const pressReleasePage = require('./pressReleasePage.graphql');
+const fragments = require('./fragments.graphql');
```

Comment:
This query file was out of date, it's been updated to match the all pages query.